source/Makefile

```diff
@@ -1,40 +1,35 @@
 .PHONY: python-html-book python-pdf-book r-html-book r-pdf-book \
 	clean
 
+python-all: python-html-book python-pdf-book
+
+r-all: r-html-book r-pdf-book
+
+Rmd_sources := $(shell python -c "import yaml; print(' '.join(yaml.load(open('_quarto.yml.template'),Loader=yaml.FullLoader)['book']['chapters']))")
+
 vector_images := $(wildcard diagrams/*.svg)
 
 diagrams := $(vector_images:.svg=.png)
 
 %.png : %.svg
 	inkscape --export-area-drawing --export-png=$@ --export-dpi=300 $<
 
-set-version-%:
+set-version-%: _quarto.yml.template _variables.yml.template
 	../scripts/set_version.py $*
 
-html-book-%:
-	$(MAKE) set-version-$*
+../%-book/index.html: set-version-% $(Rmd_sources) $(diagrams)
 	quarto render . --to html
 
-pdf-book-%:
-	$(MAKE) set-version-$*
+../%-book/resampling-with-python.pdf: set-version-% $(Rmd_sources) $(diagrams)
```

Comment:
Won't the title be `resample_with_r.pdf` for the R edition?

Reply:
Sloppy, sorry. Fixed.